docs/src/main/sphinx/develop/event-listener.rst

```diff
@@ -47,9 +50,16 @@ Example configuration file:
     custom-property1=custom-value1
     custom-property2=custom-value2
 
+.. _multiple_listeners:
+
 Multiple event listeners
 ------------------------
 
-Multiple instances of the same, or different event listeners can be
-installed and configured by setting ``event-listener.config-files``
-to a comma separated list of config files.
+Trino supports multiple instances of the same or different event listeners.
+Install and configure multiple instances by setting
+``event-listener.config-files`` in :ref:`config_properties` to a comma separated
```

Comment:
Google Style Guide and Merriam-Webster are silent on "comma-separated" vs "comma separated". Wikipedia has a page titled and using the hyphen: https://en.wikipedia.org/wiki/Comma-separated_values.  There is a site comma-separated.com whose page title is without hyphen; clearly the site is maintained by a split personality. 

I propose we standardize at Starburst on comma-separated with the hyphen.  @mosabua @m57lyra @electrum

Reply:
A search of Trino docs finds 33 "comma-separated" and 12 "comma separated". So the hyphen is already winning.

Reply:
I am a friend of the dash ;-)